.circleci/config.yml

```diff
@@ -105,6 +254,12 @@ jobs:
             VERSION=$(jq -r .version ./lerna.json)
             git checkout -b release-v${VERSION}
             git push -u origin release-v${VERSION}
+      - slack-fail:
+          branch_pattern: master
+          slack_mentions_user: << pipeline.parameters.slack_mentions_user >>
+      - slack-success:
+          branch_pattern: master
+          slack_mentions_user: << pipeline.parameters.slack_mentions_user >>
```

Comment:
If the previous `run` command fails, we'll trigger only `slack-fail`, else we'll only trigger `slack-success`.